src/main/java/Game.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+
+    List<Car> carList = new ArrayList<Car>();
+    List<Car> winnerList = new ArrayList<Car>();
+
+    public void playGame(int carNum, int roundNum) {
+        initGame(carNum);
+        for (int i = 0; i < roundNum; i++) {
+            playRound();
+        }
+        makeWinnerList();
+    }
+
+    public void initGame(int carNum){
+        for (int i = 0 ; i < carNum ; i++)
+            carList.add(new Car("Car_"+i+""));
```

Comment:
불필요한 부분은 잘 확인하시고 지워주세요!
```suggestion
            carList.add(new Car("Car_"+i));
```